src/hearings/containers/request-hearing/hearing-edit-summary/hearing-edit-summary.component.ts

```diff
@@ -647,4 +675,8 @@ export class HearingEditSummaryComponent extends RequestHearingPageFlow implemen
     }
     return false;
   }
+
+  private defaultSort(a: any, b: any): number {
```

Comment:
I don't love any in the param list. Not sure that > is very meaningful for anything except number|string - can you use that type sig?